cmd/nerdctl/image_pull_linux_test.go

```diff
@@ -145,3 +145,26 @@ CMD ["echo", "nerdctl-build-test-string"]
 	newKeyPair := newCosignKeyPair(t, "cosign-key-pair-test")
 	base.Cmd("pull", testImageRef, "--verify=cosign", "--cosign-key="+newKeyPair.publicKey).AssertFail()
 }
+
+func TestPullSoci(t *testing.T) {
+	//TODO: validate that mounts match the mounts of soci rpull
```

Comment:
No need this TODO now as rpull is working?

Reply:
I'm working on getting all the tests working in the run test working first and then transferring that functionality over to the pull test.